src/plugins/orangehrmRecruitmentPlugin/Api/CandidateAttachmentAPI.php

```diff
@@ -229,13 +229,15 @@ public function getValidationRuleForUpdate(): ParamRuleCollection
                 new Rule(Rules::STRING_TYPE),
                 new Rule(Rules::LENGTH, [!null, self::PARAM_RULE_CURRENT_ATTACHMENT_MAX_LENGTH]),
             ),
-            new ParamRule(
-                self::PARAMETER_ATTACHMENT,
-                new Rule(
-                    Rules::BASE_64_ATTACHMENT,
-                    [null, null, self::PARAM_RULE_FILE_NAME_MAX_LENGTH]
+            $this->getValidationDecorator()->notRequiredParamRule(
```

Comment:
seems you need dynamic validation :thinking: 
https://github.com/orangehrm/orangehrm/blob/5.1/src/plugins/orangehrmCorporateBrandingPlugin/Api/ThemeAPI.php#L188-L222